src/saving/SaveManagerGUI.cs

```diff
@@ -317,4 +334,30 @@ private void OnSaveListItemConfirmed(SaveListItem item)
     {
         item.LoadThisSave();
     }
+
+    private void OnFilterSelected(int index)
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        switch (index)
+        {
+            case 0:
+                saveList.Filter(SaveInformation.SaveType.Manual, true);
+                break;
+            case 1:
+                saveList.Filter(SaveInformation.SaveType.Manual);
+                break;
+            case 2:
+                saveList.Filter(SaveInformation.SaveType.AutoSave);
+                break;
+            case 3:
+                saveList.Filter(SaveInformation.SaveType.QuickSave);
+                break;
```

Comment:
Instead of a switch statement, again, just convert the index to `SaveInformation.SaveType`